.github/workflows/build.yml

```diff
@@ -1,6 +1,9 @@
 name: "Build libraries"
 
-on: [pull_request]
+on:
+  pull_request:
+    paths:
+    - 'arrow-libs/**'
```

Comment:
Hi @nomisRev !! If you want to have a clear PR, update `release/0.13.0` as well :raised_hands: